slurm_state/sacct_parser.py

```diff
@@ -391,43 +391,62 @@ def generate_job_report(
     #            remote_cmd = "/opt/slurm/bin/sacct ..."
     #        then it works. We have to hardcode the path in each cluster, it seems.
 
-    # Set the sacct command
-    # -S is a condition on the start time, 600 being in seconds
-    # -E is a condition on the end time
-    remote_cmd = f"{sacct_path} -S now-600 -E now -X --json"
-    print(f"remote_cmd is\n{remote_cmd}")
+    # Retrieve the allocations associated to the cluster
+    allocations = get_allocations(cluster_name)
```

Comment:
I fail to see the benefit of not just using 
`allocations = get_config("cluster")[cluster_name]["allocations"]`
Just like all the config values that we fetch above.

And then that function could be deleted.